opencti-platform/opencti-graphql/src/migrations/1717060854132-max-shareable-marking-in-groups.js

```diff
@@ -48,27 +48,30 @@ export const up = async (next) => {
 
   await Promise.all(groupMaxMarkingRelationCreationsPromises);
 
-  // remove platform_data_sharing_max_markings from settings
+  // set platform_data_sharing_max_markings to null in settings
   const updateQuery = {
     script: {
-      params: { null: null },
-      source: 'ctx._source.platform_data_sharing_max_markings = params.null',
+      params: { fieldToRemove: 'platform_data_sharing_max_markings' },
+      source: 'ctx._source.remove(params.fieldToRemove)',
     },
     query: {
       bool: {
         must: [
           { term: { 'entity_type.keyword': { value: 'Settings' } } },
         ],
+        should: [
+          { exists: { field: 'platform_data_sharing_max_markings' } },
```

Comment:
I guess this query works already, but I wonder why not add 
{ exists: { field: 'platform_data_sharing_max_markings' } } directly in "must" list just above like this:

```
query: {
      bool: {
        must: [
          { term: { 'entity_type.keyword': { value: 'Settings' } } },
          { exists: { field: 'platform_data_sharing_max_markings' } },
        ],
       },
    },
```

Reply:
yes you're right ! fixed :)